doc/source/learning/airship.rst

```diff
@@ -0,0 +1,175 @@
+repositories, additionally checks the documents with lint and validations via Deckhand.
+
+* Documentation
+    https://airship-pegleg.readthedocs.io/
+
+* Git repository
+    https://opendev.org/airship/pegleg
+
+
+
+Airship setup process
+----------------------
+
+To deploy using airship:
+
+.. code-block:: console
+
+  run.sh setup_nodes
+  run.sh setup_airship
+
+All the airship setup is coded in generic-deploy_airship.yml_
+
+.. _generic-deploy_airship.yml: https://github.com/SUSE-Cloud/socok8s/blob/master/playbooks/generic-deploy_airship.yml
+
+Simplifying these are its steps:
+
+1. airship-setup-deployer_:
+
+.. _airship-setup-deployer: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/airship-setup-deployer
+
+  * Prepares the environment ensuring to install repositories, system packages, openstack client packages, kubectl, helm and tiller.
+  * Starts docker
+  * Ensures VIP is present on /etc/hosts
+  * Creates /etc/openstack and copy here the clouds.yml configuration
+
+2. common-ses_:
+
+.. _common-ses: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/common-ses
+
+  * Creates the SES pools
+  * Imports SES configuration
+
+3. register-server-setup_:
+
+.. _register-server-setup: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/registry-server-setup
+
+  * Installs local images registry on the developer node
+
+4. dep-patcher_:
+
+.. _dep-patcher: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/dev-patcher
+
+  * Patches git repositories
+
+5. airship-configure-worker_:
+
+.. _airship-configure-worker: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/airship-configure-worker
+
+  * Configures sub-volumes and set them to each role
+  * Enrolls nodes depending of their group in the roles ingress, ucp, control plane, ...
+  * Builds images
+
+6. airship-configure-caasp_:
+
+.. _airship-configure-caasp: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/airship-configure-caasp
+
+  * Creates the namespaces
+
+7. airship-configure-ceph_:
+
+.. _airship-configure-ceph: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/airship-configure-ceph
+
+  * Configures ceph secrets and storage class.
+  * Applies ceph YAML
+
+8. airship-deploy-ucp_
+
+.. _airship-deploy-ucp: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/airship-deploy-ucp
+
+  * Creates the site structure, definition and manifests (profile, software, secrets)
+  * Creates pod scale profile
+  * Builds images (Armada, Shipyard, Airflow, Deckhand, Pegleg)
+  * Deploys Armada
+  * Sets pegleg parameters, collects ucp site manifest and copy to Armada.
+  * Deploys Armada-UCP
+
+9. airship-deploy-osh_ :
+
+.. _airship-deploy-osh: https://github.com/SUSE-Cloud/socok8s/tree/master/playbooks/roles/airship-deploy-osh
+
+  * Creates the site configuration using Pegleg
+  * Sends it to Shipyard
```

Comment:
This is a great summary. Thanks @ilausuch !

Reply:
Thank you :)